pkg/virt-controller/watch/vmi.go

```diff
@@ -154,22 +155,32 @@ func NewVMIController(templateService services.TemplateService,
 	topologyHinter topology.Hinter,
 ) *VMIController {
 
+	sriovVmiCopyAnnotExecPool := executor.NewRateLimitedExecutorPool(
```

Comment:
@EdDev IIUC this backoff should allow a check every 10ms with a mac of 100 steps. Do you feel that we should usea a smaller resolution?